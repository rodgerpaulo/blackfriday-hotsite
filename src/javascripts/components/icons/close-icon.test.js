import React from 'react';
import CloseIcon from './close-icon';
import renderer from 'react-test-renderer';

test('Render Icons with default state', () => {
  const component = renderer.create(<CloseIcon />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
