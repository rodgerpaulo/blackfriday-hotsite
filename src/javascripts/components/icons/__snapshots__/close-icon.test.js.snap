// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Icons with default state 1`] = `
<svg
  className=""
  color="#505050"
  fill="#505050"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fill="none"
    fillRule="evenodd"
    opacity=".54"
  >
    <g
      fill="#505050"
      transform="translate(-415 -99)"
    >
      <g
        transform="translate(416 99)"
      >
        <polygon
          points="19 6.4 17.6 5 12 10.6 6.4 5 5 6.4 10.6 12 5 17.6 6.4 19 12 13.4 17.6 19 19 17.6 13.4 12"
        />
      </g>
    </g>
  </g>
</svg>
`;
